### PR TITLE
i18n(fr): add `resources/plugins`

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

This PR adds the french translation of the `resources/plugins` page (which is mostly moved content from the previous showcase) and also updates the link to it in `reference/configuration`.

### CODE REVIEW

File: docs/src/content/docs/fr/resources/plugins.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Modules d'extension et intégrations
```
Comment (delucis) [inline docs/src/content/docs/fr/resources/plugins.mdx:2]: Non blocking, but noting that this title is quite long, causing it to wrap when it’s the current page:

sidebar with page title

It’s an accurate translation though, so I’m happy if you are. I guess one other option could be “Modules d'extension et outils”?

**Review by delucis — APPROVED** [review]
LGTM! Left one comment but up to you whether it needs any action or not.

File: docs/src/content/docs/fr/resources/plugins.mdx (hunk 1 above)
Comment (HiDeoo) [reply docs/src/content/docs/fr/resources/plugins.mdx]: Great catch, and based on the content, your suggestion is totally valid so I went with it 👍

**Comment by HiDeoo** [comment]
> LGTM! Left one comment but up to you whether it needs any action or not.

As I went with your suggestion, I am using the previous approval to merge the PR without a new review 🫣

**Pull request merged**